### PR TITLE
W-gan

### PR DESCRIPTION
**Pull request by husonchen**
WassersteinGAN in mxnet.

### CODE REVIEW

**Comment by mli** [comment]
Job PR-272-1 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/1/index.html 
Code coverage of this PR:  vs. Master:

**Comment by husonchen** [comment]
@zhreshold I have removed importing six and modified LSUN downloader.

**Comment by mli** [comment]
Job PR-272-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/3/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-272-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/4/index.html 
Code coverage of this PR:  vs. Master:

File: scripts/datasets/lsun.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function, division
+import argparse
+import json
+from os.path import join
+
+import subprocess
```
Comment (zhreshold) [inline scripts/datasets/lsun.py]: this is no longer required

**Comment by zhreshold** [comment]
Since it's a script project, I think it's good to be merged
@husonchen could you add some descriptions to the dependencies such as mxboard? We need to make sure there's no error if a user wants to run the script for the first time.

**Comment by mli** [comment]
Job PR-272-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-272-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/7/index.html 
Code coverage of this PR:  vs. Master:

**Comment by husonchen** [comment]
@zhreshold I have improved readme file and import statements.

**Comment by zhreshold** [comment]
@hetong007 @zhanghang1989 This is the PR from summer camp, do you have further comments?

**Comment by mli** [comment]
Job PR-272-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/8/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-272-9 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/9/index.html 
Code coverage of this PR:  vs. Master:

**Comment by zhreshold** [comment]
@zhanghang1989 If you don't have further comments I will merge this tomorrow.

**Comment by mli** [comment]
Job PR-272-10 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-272/10/index.html 
Code coverage of this PR:  vs. Master:

**Comment by zhreshold** [comment]
@husonchen This is now merged! Thanks for the contribution

**Pull request merged**